tests/test_anonfile.py

```diff
@@ -38,73 +39,116 @@ def remove_file(file: str) -> None:
     Path(file).unlink() if Path(file).exists() else None
 
 
-class TestAnonFileLibrary(unittest.TestCase):
+class MockData:
```

Comment:
I would separate the `MockData` class from the unit tests for the following reasons:

1. keeps the test file itself small and contained to the tests in questions
2. makes it possible to import the `MockData` into other unit tests*

---

*technically speaking, you could still import it from `test_anonfile` but this would look rather confusing without any further inside the whole test project

Reply:
I thought this was a cleaner approach, so I changed the test to reflect this comment